a-short-introduction-to-kubernetes/index.md

```diff
@@ -0,0 +1,372 @@
+---
+title: 'Introduction to Kubernetes'
+description: 'A short introduction to Kubernetes in a Directus context'
+author:
+  name: 'Mikke Schirén'
+  avatar_file_name: 'mikke.png'
+---
+
+## Introduction
+
```

Comment:
Let's open with the problem. What happens without K8S and therefore why does it need to exist?